lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -730,6 +729,21 @@ def juggle_axes(xs, ys, zs, zdir):
     else:
         return xs, ys, zs
 
+def juggle_axes_vec(xyz, zdir):
```

Comment:
Can you make this a private function and (if other devs agree) deprecate `juggle_axes`?  Similarly for `rotate_axes`.  Let's take advantage of the opportunity to reduce the public API...